docs/REST API/rest-api.md

```diff
@@ -3891,6 +3891,7 @@ Add a configuration profile to enforce custom settings on macOS and Windows host
 | ------------------------- | -------- | ---- | ------------------------------------------------------------------------------------------------------------- |
 | profile                   | file     | form | **Required.** The .mobileconfig (macOS) or XML (Windows) file containing the profile. |
 | team_id                   | string   | form | _Available in Fleet Premium_ The team ID for the profile. If specified, the profile is applied to only hosts that are assigned to the specified team. If not specified, the profile is applied to only to hosts that are not assigned to any team. |
+| labels                   | array     | form | An array of labels to filter hosts in a team (or no team) that should get a profile. |
```

Comment:
Also, I think `label` will be premium only because the feature is paid. Let's call this out in the API docs. Not sure how we do that elsewhere in the API docs.

Reply:
Added "_Available in Fleet Premium_" to description.